pkg/filter/http/httpproxy/routerfilter.go

```diff
@@ -119,8 +118,8 @@ func (f *Filter) Decode(hc *contexthttp.HttpContext) filter.FilterStatus {
 	endpoint := clusterManager.PickEndpoint(clusterName, hc)
 	if endpoint == nil {
 		logger.Debugf("[dubbo-go-pixiu] cluster not found endpoint")
-		bt, _ := json.Marshal(contexthttp.ErrResponse{Message: "cluster not found endpoint"})
-		hc.SendLocalReply(http.StatusServiceUnavailable, bt)
+		errResp := contexthttp.ServiceUnavailable.WithError(fmt.Errorf("endpoint not found"))
```

Comment:
if the format func is not used when creating a new err, use `errors.New` instead of `fmt.Errorf`. `fmt.Errorf` only used when need format

```suggestion
		errResp := contexthttp.ServiceUnavailable.WithError(errors.New("endpoint not found"))
```